Add tips section for calendar components (#6354)
yamada-ui/yamada-ui: www/contents/components/(components)/date-picker.mdx

<DatePicker />
```

:::tip

> If you want a calendar that stays visible on the page for viewing or selection, use [Calendar](/docs/components/calendar).

Reviewer: Let's keep this tip, but mention viewing or selecting a date from an always-visible calendar and align the wording with the other tips.

```suggestion
If you want users to view or select a date from an always-visible calendar, use [Calendar](/docs/components/calendar).
```

> If you want to keep dates always visible on the page so they can be checked or selected, use [Calendar](/docs/components/calendar).

Reviewer 2: This is understandable, but I think it reads more naturally if the distinction is framed as `Calendar` being the always-visible option on the page.

```suggestion
If you want a calendar that stays visible on the page for viewing or selection, use [Calendar](/docs/components/calendar).
```

Reviewer 3: I think the wording you suggested looks good.
